src/os/mapcontainer.js

```diff
@@ -776,6 +776,8 @@ class MapContainer extends EventTarget {
     // Map
     controls.addControl(genMapGrp, 1, 'Draw Geometry',
         [KeyCodes.SHIFT, '+'], [Controls.MOUSE.LEFT_MOUSE, '+', Controls.FONT.ALL]);
+    controls.addControl(genMapGrp, 1, 'Measure',
+        [platformModifier, '+', KeyCodes.SHIFT, '+'], [Controls.MOUSE.LEFT_MOUSE, '+', Controls.FONT.ALL]);
```

Comment:
Can we remove the drag part? If you mouse down and drag the interaction doesn't do anything, unless we fix that in the polygon interaction. We have drag on Draw Geometry for the box/circle interaction, but it doesn't work the same with Polygon and Measure.

Reply:
Yeah I could see this going either way. I see that as "move the mouse around", not necessarily just drag. Some sort of an "click and hold" indicator would suggest dragging to me.